src/pages/home/report/ReportActionCompose.js

```diff
@@ -144,7 +144,9 @@ class ReportActionCompose extends React.Component {
             },
             maxLines: props.isSmallScreenWidth ? CONST.COMPOSER.MAX_LINES_SMALL_SCREEN : CONST.COMPOSER.MAX_LINES,
             value: props.comment,
-            conciergePlaceholderRandomIndex: _.random(this.props.translate('reportActionCompose.conciergePlaceholderOptions').length - 1),
+
+            // If we are on a small width device then don't show last 3 items from conciergePlaceholderOptions
+            conciergePlaceholderRandomIndex: _.random(this.props.translate('reportActionCompose.conciergePlaceholderOptions').length - (this.props.isSmallScreenWidth ? 4 : 1)),
```

Comment:
nab, but should we have had moved `4` to a constant? I am not fixated because we also have a comment just above the line.

Reply:
I think it's fine, this does not seem to used somewhere else, and the comment makes it easy to understand